feconf.py

```diff
@@ -45,7 +45,7 @@
 ISSUES_DIR = (
     os.path.join(EXTENSIONS_DIR_PREFIX, 'extensions', 'issues'))
 INTERACTIONS_DIR = (
-    os.path.join(EXTENSIONS_DIR_PREFIX, 'extensions', 'interactions'))
+    os.path.join('extensions', 'interactions'))
```

Comment:
Hm why is this inconsistent? It's starting to get a bit muddy, maybe we should rename/clarify the point of EXTENSIONS_DIR_PREFIX

Reply:
Please add comments inline to this file to explain, for devs who look at it.

Reply:
Done.